lib/ReactViews/Story/StoryPanel/StoryBody.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Story } from "../Story";
+import parseCustomHtmlToReact from "../../Custom/parseCustomHtmlToReact";
+import styled from "styled-components";
+
+const StoryContainer = styled.div<{ isCollapsed: boolean }>`
+  max-height: ${props => (props.isCollapsed ? 0 : "100px")};
+  @media (min-height: 700px) {
+    max-height: ${props => (props.isCollapsed ? 0 : "200px")};
+  }
+  @media (min-height: 900px) {
+    max-height: ${props => (props.isCollapsed ? 0 : "400px")};
+  }
```

Comment:
For the media query, can we use `${p => p.theme.sm}` (from variables [defined here](https://github.com/TerriaJS/terriajs/blob/edbf5deab37cd7adf6c957b76ae34c4a1e39c909/lib/Sass/common/_variables.scss#L158-L161)).

Reply:
thanks, will change

Reply:
actually no we can't, this is min-height not max-width. Ie it is a special use case to figure out how high we want the story to be based on the vertical real estate of the screen

Reply:
ah ok got it :+1: